### ISSUE TITLE
Duplicate handlers do not cause a compilation error

### ISSUE THREAD

**Issue by szagi3891** [poem-web/poem#126]
## Expected Behavior
I would expect that the macro would not allow this code to compile

## Actual Behavior
When we have duplicate handlers, the code compiles.

```
    #[oai(method = "get", path = "/fun1")]
    async fn fun1(&self) -> PlainText<String> {
        PlainText("response from fun1".into())
    }

    #[oai(method = "get", path = "/fun1")]
    async fn fun2(&self) -> PlainText<String> {
        PlainText("response from fun2".into())
    }
```

## Steps to Reproduce the Problem

1. You have to clone https://github.com/szagi3891/poem-test
2. And then run with cargo run

## Specifications

- Version: poem = "1.0.38", poem-openapi = "1.0.41"
- Platform: MacOs
- Subsystem:

**Comment by sunli829** [comment]
I cannot detect duplicate paths at compile time, but I can panic at runtime.

**Comment by szagi3891** [comment]
@sunli829 Is it possible to make this panic happen when starting the application?

**Comment by sunli829** [comment]
Yes. If there are duplicate paths, a panic will occur  when starting the application. 🙂

**Comment by sunli829** [comment]
Released in `v1.2.6`

**Comment by szagi3891** [comment]
Something is still not working. Here I have an example of such duplicate handlers:
https://github.com/szagi3891/poem-test/blob/master/src/main.rs#L142

When I run cargo run, the server gets up (no panic). And it should panic with a message that "GET /fun1" is duplicated

**Comment by sunli829** [comment]
This can be detected in the macro, I forgot to do this. 🙂

**Comment by sunli829** [comment]
Fixed in `v1.2.7`

**Comment by szagi3891** [comment]
Fantastic. It works perfectly. Thank you very much :)
